Dockerfile

```diff
@@ -8,4 +9,4 @@ RUN make
 
 FROM scratch
```

Comment:
how about this layer?

Reply:
https://github.com/code-ready/gvisor-tap-vsock/pull/6#discussion_r539984339